sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -776,6 +776,23 @@ abstract class TypeChecks[RET] {
     }.mkString(", ")
   }
 
+  private def tagCustomMessage(
+    groupedByType: Map[DataType, Set[String]],
+    meta: RapidsMeta[_, _, _]
+    ): Map[DataType, Set[String]] = {
+    groupedByType.filterKeys {
```

Comment:
Seems can not check nested types like `Array(timestamp)`

Reply:
done